core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -670,14 +669,8 @@ private static TabletMetadata getRootMetadata(ClientContext ctx,
       case EVENTUAL:
         return getRootMetadata(zkRoot, ctx.getZooCache());
       case IMMEDIATE:
-        ZooReader zooReader = ctx.getZooReader();
```

Comment:
Not related to this change - but do we still want to allow an eventual read of root metadata?

Reply:
Looking at how this it used, I think this code could be cleaned up alot.  The eventual read is only for getting the root tablet location.  It would probably be better to just move it to the tablet location cache code and remove it from ample, simplifying ample.  I can do that in a follow on PR, I also see a problem w/ the cache invalidation that would be cleaner to fix if the code were consolidated.

Reply:
Currently working on changes that build on top of these changes to remove the EVENTUAL consistency level from Ample.  But will post that as another PR.

Reply:
While working on changes related to removing EVENTUAL consistency level, identified 4669 as a prerequisite for those changes